src/Magnum/AbstractFramebuffer.h

```diff
@@ -117,19 +117,19 @@ enum class FramebufferTarget: GLenum {
     /** Frambebuffer reading target */
     #ifndef MAGNUM_TARGET_GLES2
     Read = GL_READ_FRAMEBUFFER,
-    #elif !defined(MAGNUM_TARGET_WEBGL)
+    #elif defined(CORRADE_TARGET_APPLE)
     Read = GL_READ_FRAMEBUFFER_APPLE,
```

Comment:
Note: I'm not *entirely* certain if this is correct. Maybe the best solution here is to look at the supported extension and try and figure out the correct framebuffer binding from there. Only supporting multi-sampling on iOS seems to me like a pretty big assumption.